test/integration/targets/vmware_guest_find/aliases

```diff
@@ -0,0 +1,4 @@
+posix/ci/cloud/vcenter
+cloud/vcenter
+skip/python3
+posix/ci/group1
```

Comment:
This test shouldn't have `posix/ci/group1` since it has `posix/ci/cloud/vcenter`.